Document how to close emitting side once we are done using subscription queries. (#107)
AxonIQ/reference-guide: implementing-domain-logic/query-handling/dispatching-queries.md

4. The `SubscriptionQueryResult#handle(Consumer<? super I>, Consumer<? super U>)`
 method gives us the possibility to subscribe to the `initialResult` and the `updates` in one go. 
If we want more granular control over the results, we can use the `initialResult()` and `updates()` methods on the query result.

> 5. As `queryUpdateEmitter` will continue to emit updates even when there are no subscribers, we need to notify emitting side once we are no longer interested in receiving updates.

Reviewer: ```suggestion
5. As the `queryUpdateEmitter` will continue to emit updates even when there are no subscribers, we need to notify the emitting side once we are no longer interested in receiving updates.
```

> Failing to do so can result with hanging infinitive streams and eventually memory leak.

Reviewer: ```suggestion
Failing to do so can result in hanging infinitive streams and eventually a memory leak.
```

> Once we are done with using subscription query, we need to close used resource. We can do that in `doFinally` hook. 

Reviewer: ```suggestion
Once we are done with using subscription query, we need to close the used resource. We can do that in `doFinally` hook. 
```

> As an alternative to the `doFinally` hook, there is the Flux.using API. This is synonymous

Reviewer: ```suggestion
As an alternative to the `doFinally` hook, there is the `Flux#using` API. This is synonymous
```

> with the try-with-resource Java API:

Reviewer: ```suggestion
to the try-with-resource Java API:
```
